lib/iris/fileformats/netcdf.py

```diff
@@ -1958,7 +1949,11 @@ def set_packing_ncattrs(cfvar):
             # Explicitly assign the fill_value, which will be the type default
             # in the case of an unmasked array.
             if packing is None:
-                fill_value = cube.fill_value
+                if hasattr(cube.lazy_data(), 'fill_value'):
+                    fill_value = cube.lazy_data().fill_value
+                else:
+                    fill_value = None
+
```

Comment:
Iris 1.13 has
```
            if packing is None:
                fill_value = cube.lazy_data().fill_value
                dtype = cube.lazy_data().dtype.newbyteorder('=')
```
Does that mean we could get the `fill_value` from cube.lazy_data() when we were still using biggus?